apps/vaporgui/ErrorReporter.cpp

```diff
@@ -197,7 +197,7 @@ string ErrorReporter::GetSystemInformation()
 	ret += "OS: " + string(info.sysname) + " " + string(info.release) + " " + string(info.version) + "\n";
 	ret += "Distro:\n";
 	char buffer[128];
-    	FILE *pipe = popen("lsb_release", "r"), pclose;
+    	FILE *pipe = popen("lsb_release", "r");
```

Comment:
pipe is not being closed

Reply:
That's a bug of the code. I intended to only address warnings in this PR, unless you think I should fix bugs along the way.

Reply:
since this bug is so easy to fix, I fixed via commit 499f0ba